src/engines/nwn2/roster.h

```diff
@@ -27,6 +27,8 @@
 
 #include <list>
 
+#include "src/aurora/types.h"
```

Comment:
(Minor nitpick: please put the aurora include after the common include (but still separated by a newline))